applications/tari_merge_mining_proxy/src/main.rs

```diff
@@ -340,14 +363,47 @@ fn handle_connection(
         .map_err(|e| MmProxyError::OtherError(format!("Error reading from stream {}", e)))?;
     let date = Local::now();
     let request_string = stringify_request(&buffer[..]);
+    debug!(target: LOG_TARGET, "Request: {}", request_string);
     let request_type = get_request_type(&buffer[..])?;
     let url_part = get_url_part(&buffer[..])?;
     if request_type.starts_with("GET") {
-        debug!(target: LOG_TARGET, "Handling Method: {}", url_part);
+        info!(target: LOG_TARGET, "Handling GET Method: {}", url_part);
```

Comment:
```suggestion
        debug!(target: LOG_TARGET, "Handling GET Method: {}", url_part);
```
This should not be info
this should be debug. 

Info messages are messages that the non technical user might want to see, like accepted new block, mined new block. 
Not handling metod etc.

Reply:
Done